### DISCUSSION TITLE
证券时报：谨防烂尾楼盘停供风险扩散

### DISCUSSION THREAD

**Discussion by WeNeedHome** [WeNeedHome/SummaryOfLoanSuspension#36] [General]
http://www.stcn.com/stock/djjd/202207/t20220713_4735107.html

![image](https://user-images.githubusercontent.com/109166931/178672038-aa67d21c-04b8-46e1-b8e5-2ba302e079f5.png)

**Comment by AntEffect** [comment]
又要开始拉一批打一批，全网封禁一批了吗

**Reply by liuwenwu520** [reply]
网上都开始删除新闻了，中央政府也不出来表态，这件事真的要被压下来了呀

**Reply by humSuperman** [reply]
不要寄希望他们表态，就是他们做的怪。能跑就跑

**Comment by ZQlby** [comment]
风险是重中之重，人不是，钱不是，房不是🌝

**Comment by xujiyou** [comment]
吃进去的钱，我就没见过有吐出来的时候

**Comment by jwwb681232** [comment]
根本原因是制度出了问题（同时并不代表西方制度没问题）

**Reply by rechardlc** [reply]
一千多年的专制制度~改不了的~

**Reply by feidao158** [reply]
王侯将相 宁有种乎

**Reply by AntEffect** [reply]
> 王侯将相 宁有种乎

为什么要有王侯将相？人人平等不好吗？

**Reply by yjzfun** [reply]
独裁没办法，自古以来独裁的下场就是被推翻（我是说封建王朝）

**Reply by ineedabitchlikeu** [reply]
骨子里的观念 百姓逆来顺受 当官的权势滔天

**Comment by wjbviapp** [comment]
现房交付就可以了，期房只会烂下去

**Reply by qq912276337** [reply]
还有高杠杆 6个钱包 割死韭菜

**Comment by weikangzhao** [comment]
金融系统失去的不光是这些停贷业主的钱，失去的是全国人民的信心，金融业信心和共识比金子还要宝贵。

**Comment by zhaokai5520** [comment]
信心比黄金更重要

**Comment by awmix** [comment]
资本开始给房地产洗地了...

**Comment by humSuperman** [comment]
在中国为官，要相信后来人的智慧，不要烂在我位子上就行。

**Comment by n1248** [comment]
对于他们，烂尾楼盘不是风险，烂尾楼盘停供是风险

**Comment by astramare** [comment]
再苦一苦百姓🐶

**Reply by Leon0312** [reply]
天下苦百姓久矣

**Comment by bobqq33** [comment]
证券时报是证监会和银保监合办的，这波等于是金融系统担心断供给银行带来太大压力，向政府喊话要求政府保复工保交房

**Comment by helloworld001111** [comment]
当年忠贞为国酬，何曾怕断头？如今天下红遍，江山靠谁守？业未就，身躯倦，鬓已秋。你我之辈，忍将夙愿，付与东流.
--毛泽东绝笔

**Comment by bobo418** [comment]
经济好的时候是农民，经济不好的时候农民就是背锅的

**Comment by bobo418** [comment]
解决不了事情，就解决人

**Comment by iiouoii** [comment]
烂尾楼的风险不用防，停贷的你们是谨防的对象

**Reply by unlockth** [reply]
以前国家穷用房地产拉动经济，玩空手套白狼可以理解。现在国家有钱了，也是时候把经济转移到工业上，房地产拉动经济已经不重要了，就应该改革让房产的预售机制取消，做真正有利于人民的事情。
